src/components/select/select.tsx

```diff
@@ -194,14 +210,14 @@ export class Select implements FormComponent {
     return this.renderer.getNativeInputId();
   }
 
-  private setDisabled() {
-    this.options.forEach(option => {
-      option.disabled = this.disabled;
-    });
-  }
+  //   private setDisabled() {
+  //     this.options.forEach(option => {
+  //       option.disabled = this.disabled;
+  //     });
+  //   }
```

Comment:
Let's remove the commented code

Reply:
This is was asked to you in commit f8fff15
If I should remove that method or not. That method provokes the issue with the disable/enable option.
Should I remove the comment or remove the whole method?

Reply:
I mean removing the block of commented code, of course. Not uncommenting it.

Reply:
@ncardeli I think it should be better if we leave the method commented, maybe it will be necessary in the future. Do you suggest to remove it (delete it from the code)?

Reply:
Yes, if we want to recover it, we have the history that Git stores for us.

Reply:
Ok, I will remove it now